tree/dataframe/inc/ROOT/RDF/ActionHelpers.hxx

```diff
@@ -238,6 +256,13 @@ public:
 
    void Finalize();
 
+   // Helper functions for RMergeableValue
+   void * GetMergeableValue()
+   {
+      RMergeableValue<Hist_t>* mergeableptr{new RMergeableHisto1D<Hist_t>{*fResultHist}};
```

Comment:
here and in FillParHelper, can we avoid copying the histogram? the RResultPtr stores it in a shared_ptr after all. In large analyses with a lot of large histograms, this could be a non-negligible memory cost (runtime cost too, a little bit).

Reply:
Fair point, I will take a look at that!

Reply:
After our discussions we decided this is not easily solvable for the moment. The best solution would be a move operation for the histogram imho